weed/s3api/s3api_internal_lifecycle_test.go

```
@@ -117,3 +117,33 @@ func TestLifecycleDelete_RejectsEmptyRequest(t *testing.T) {
+	s := &S3ApiServer{}
+	cases := []string{
+		"",
+		".uploads",
+		".uploads/",
+		".uploads/.",
+		".uploads/..",
+		".uploads/u1/extra",
+	}
+	for _, path := range cases {
+		t.Run(path, func(t *testing.T) {
+			resp, err := s.LifecycleDelete(nil, &s3_lifecycle_pb.LifecycleDeleteRequest{
```

golangci-lint (2.12.1)

[error] SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use

(staticcheck)